WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -75,11 +97,16 @@
 
 <script>
   $('#<%= @past_selected ? "past" : "present" %>').button('toggle');
+  $('#<%= @recent_selected ? "recent" : ""%>').button('toggle');
 
```

Comment:
I agree, it seems weird to be virtually clicking 2 buttons when the page loads. Can this be changed to only click 1?